spec/features/resource_manager_manages_schedules_spec.rb

```diff
@@ -95,11 +121,14 @@ def then_they_are_told_that_the_schedule_has_been_created
     @page = Pages::EditUser.new
     expect(@page).to have_flash_of_success
   end
-
-  def then_they_are_told_that_the_schedule_has_been_updated
-    @page = Pages::EditUser.new
-    expect(@page).to have_flash_of_success
-  end
+  alias_method(
```

Comment:
Maybe just drop the parentheses and have the first argument on the same line as `alias_method` and each successive argument on a new line? Seems a little neater than this style to my eyes. But i'm not overly fussed on this.